Mathlib/Algebra/Module/Equiv/Basic.lean

```diff
@@ -323,10 +322,10 @@ def addMonoidHomLequivNat {A B : Type*} (R : Type*) [Semiring R] [AddCommMonoid
     where
   toFun := AddMonoidHom.toNatLinearMap
   invFun := LinearMap.toAddMonoidHom
-  map_add' := by intros; ext; rfl
-  map_smul' := by intros; ext; rfl
-  left_inv := by intro f; ext; rfl
-  right_inv := by intro f; ext; rfl
+  map_add' := fun _ _ ↦ rfl
```

Comment:
Does `map_add' _ _ := rfl` work? Similar for the other fields.

Reply:
Yes, it does. Done, thanks.